l5kit/l5kit/data/__init__.py

```diff
@@ -1,9 +1,15 @@
 from .combine import get_combined_scenes
-from .filter import filter_agents_by_frame, filter_agents_by_frames, filter_agents_by_labels, get_agent_by_track_id
+from .filter import (
+    filter_agents_by_frame,
+    filter_agents_by_frames,
+    filter_agents_by_labels,
+    filter_tr_faces_by_frames,
+    get_agent_by_track_id,
+)
 from .labels import LABEL_TO_INDEX, LABELS
 from .local_data_manager import DataManager, LocalDataManager
-from .map import load_semantic_map
-from .zarr_dataset import AGENT_DTYPE, FRAME_DTYPE, SCENE_DTYPE, ChunkedStateDataset
+from .map import proto_to_semantic_map
+from .zarr_dataset import AGENT_DTYPE, FRAME_DTYPE, SCENE_DTYPE, TR_FACES_DTYPE, ChunkedStateDataset
```

Comment:
I would not use too short names here, `TR` is not very telling. Prefer `TRAFFIC_LIGHTS_DTYPE`, what do you think?

Reply:
Although I agree, that will make lot of strings too long for the linter :(

Reply:
@lucabergamini I do not mind to have another line of code for a clearer data type definition